build.sbt

```diff
@@ -85,15 +86,25 @@ lazy val hq = (project in file("hq"))
     testOnly in Test := (testOnly in Test).dependsOn(startDynamoDBLocal).evaluated,
     testQuick in Test := (testQuick in Test).dependsOn(startDynamoDBLocal).evaluated,
     testOptions in Test += dynamoDBLocalTestCleanup.value,
+
+    serverLoading in Debian := Some(Systemd),
+    debianPackageDependencies := Seq("openjdk-8-jre-headless"),
+    maintainer := "Security Team <devx.sec.ops@guardian.co.uk>",
+    packageSummary := "Brief description",
+    packageDescription := """Slightly longer description""",
     riffRaffPackageType := (packageBin in Debian).value,
     riffRaffUploadArtifactBucket := Option("riffraff-artifact"),
     riffRaffUploadManifestBucket := Option("riffraff-builds"),
+
+    riffRaffAwsRegion := "eu-west-1", // Needed for GHA while we improve how the plugin works here.
```

Comment:
This shouldn't be needed as the assume-aws-role action [sets some env vars already](https://github.com/guardian/actions-assume-aws-role/blob/main/src/main.ts#L39-L40). Curious to know how change performs without this.

Reply:
Interesting, will take another look. I was seeing errors on this when not explicitly set - about failure to retrieve from instance metadata.

Reply:
Ah. I _think_ this is the result of [this call](https://github.com/guardian/sbt-riffraff-artifact/blob/master/src/main/scala/com/gu/riffraff/artifact/RiffRaffArtifact.scala#L65), specifically `Regions.getCurrentRegion`?

This is reported incorrectly by the plugin, it is a warning, not an error.